src/test/java/edu/harvard/iq/dataverse/api/AdminIT.java

```diff
@@ -895,4 +896,15 @@ private String createTestNonSuperuserApiToken() {
         createUserResponse.then().assertThat().statusCode(OK.getStatusCode());
         return UtilIT.getApiTokenFromResponse(createUserResponse);
     }
+
+    @ParameterizedTest
+    @ValueSource(booleans={true,false})
+    public void testChangeSuperUserStatus(Boolean status) {
```

Comment:
Can you please change this to "testSetSuperuserStatus" as well?

Reply:
@pdurbin done